packages/runtime/src/useCases/transport/publicRelationshipTemplateReferences/PublicRelationshipTemplateReferenceMapper.ts

```diff
@@ -0,0 +1,8 @@
+import { PublicRelationshipTemplateReference } from "@nmshd/transport";
+import { PublicRelationshipTemplateReferenceDTO } from "../../../types/transport/PublicRelationshipTemplateReferenceDTO";
+
+export class PublicRelationshipTemplateReferenceMapper {
+    public static toPublicRelationshipTemplateReferencDTO(tagList: PublicRelationshipTemplateReference): PublicRelationshipTemplateReferenceDTO {
```

Comment:
The input parameter `tagList` doesn't seem to be named appropriately in this context.